.azure/windows-installhs-cabal.yml

```diff
@@ -17,21 +18,21 @@ jobs:
     displayName: Install GHC
   - bash: |
       source .azure/windows.bashrc
-      stack install cabal-install --stack-yaml $(YAML_FILE)
+      stack install Cabal-3.0.0.0 cabal-install-3.0.0.0 --stack-yaml $(YAML_FILE)
```

Comment:
Maybe document this why we are using stack to install cabal? Why not choco?

Reply:
I checked it and it seems choco is available for azure windows agents so it would be doable.
But i would left it to another pr, if you dont mind (adding a TODO)

Reply:
adding a todo is fine for me!